content/docs/configuration.md

```diff
@@ -551,6 +551,13 @@ settings.
 If you do not want to give us `admin` permission, you can update the project
 settings manually in Copr based on the guidance Packit Service gives.
 
+In order to trigger a build in a custom Copr project through an action in a GitHub repo
+(like opening a pull request, commenting a pull request with '/packit copr-build', 
+releasing a new version etc.), the GitHub repo has to be listed in the 
+**Packit forge project allowed** field in the **Copr project settings**.
```

Comment:
```suggestion
When using a custom Copr project (by specifying `project` and `owner`), the GitHub repo has to be listed in the
**Packit forge project allowed** field in the **Copr project settings** so that the Copr builds can be actually run.
```

Reply:
I would probably use wording similar to this and also move it right after the config options so that the paragraph `builder` permission would follow this.